.pubnub.yml

```diff
@@ -66,45 +66,52 @@ sdks:
                     minimum-os-version:
                       - Windows Vista Ultimate
                     maximum-os-version:
-                      - Windows 10 Home
+                      - Windows 11 Home
                     target-architecture:
                       - x86
                       - x86-64
               requires:
+                -
+                  name: kotlin
+                  min-version: 2.0.21
```

Comment:
we use 2.0.21, but we're compatible with at least 1.8.0 and above, which I think was the point to capture this information here